Automattic/sensei
# Fix Course Wizard spacing when using the latest Gutenberg

Resolves #6939

## Proposed Changes

This PR fixes the modal content height issue on the latest Gutenberg version while aiming to not break the layout in the older versions.

## Testing Instructions


1. Install the Gutenberg plugin.
2. Go to Sensei LMS -> Courses -> New Course.
3. Make sure the course wizard has no blank space at the bottom.
4. Click `Continue` and make sure the next step looks good as well.

## Pre-Merge Checklist
